spec/index.html

```diff
@@ -598,6 +598,37 @@ <h4>INSERT DATA</h4>
 &lt;http://example/book1&gt; dc:title "Fundamentals of Compiler Design" .
 &lt;http://example/book1&gt; ns:price 42 .
 </pre>
+
+          <p><strong>Example 3: Adding a quoted triple</strong></p>
+          <p><span class="doc-ref" id="example_insert_quoted"></span>
+              When inserting triples containing <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triples</a>,
+              these quoted triples are not automatically inserted as <a data-cite="RDF12-CONCEPTS#dfn-asserted-triple">asserted triples</a>.
+              In other words, when executing the <code><a href="#insertData">INSERT DATA</a></code> operation below,
+              the default graph will <em>not</em> contain the triple <code>:bob :age 23</code>.
+          </p>
+          <pre class="query nohighlight">PREFIX : &lt;http://www.example.org/&gt;
+INSERT DATA
+{ :alice :claims &lt;&lt; :bob :age 23 &gt;&gt; . }
+</pre>
+          <p>Data after:</p>
```

Comment:
It is not clear what the data before was. It is the empty graph but could be read as same as example 2 where the initial graph had triples in it.